docs/teachertool/catalog.json

```
@@ -12,6 +12,106 @@
+        },
+        {
+            "id": "01A65046-D4BE-44B6-8273-7A878A71B3D0",
+            "use": "point_bool_check",
+            "template": "Check that an LED point is lit on the LED screen",
+            "description": "At point 'row', 'col' on the LED screen, the LED is lit",
+            "hideInCatalog": true,
+            "docPath": "/teachertool"
+        },
+        {
+            "id": "E3286B6D-5BEB-43EC-B246-C956AAC34C3E",
+            "use": "unplot_vars_used",
+            "template": "Unplot a point on the LED screen at 'col', 'row'",
+            "description": "The unplot block is used to unset the LED at the position 'col', 'row'",
+            "hideInCatalog": true,
+            "docPath": "/teachertool"
+        },
+        {
+            "id": "46AFEDA3-35CB-4041-BEC4-53C0D29E57BA",
+            "use": "col_add_num",
+            "template": "Perform addition between the 'col' variable and a number",
+            "description": "Arithmetic, paritciularly addition, is performed between the 'col' variable and a number",
+            "hideInCatalog": true,
+            "docPath": "/teachertool"
+        },
+        {
+            "id": "6BA59761-9464-4C8D-9D8F-E8985273FB92",
+            "use": "plot_vars_used",
+            "template": "Plot a point on the LED screen at position 'col' + 1, 'row'",
+            "description": "The plot block sets the LED at the position 'col' + 1, 'row'",
+            "hideInCatalog": true,
+            "docPath": "/teachertool"
+        },
+        {
+            "id": "1B331451-081E-4F26-9CB6-F5A649432BF9",
+            "use": "repeat_loop_nested_conditions",
+            "template": "Repeat loop holds all main program logic",
+            "description": "Checks that the repeat loop has two if conditions, and everything within those conditions looks as expected",
+            "hideInCatalog": true,
+            "docPath": "/teachertool"
+        },
+        {
+            "id": "A4F05FAB-72F2-48F5-8E33-7AE2CBB6CD43",
+            "use": "forever_nested_repeat_loop",
+            "template": "Forever loop nests a repeat loop that holds all main program logic",
+            "description": "Checks that the forever loop and all of its children exist within the loop. Essentially checks that the whole program looks as expected",
+            "hideInCatalog": true,
+            "docPath": "/teachertool"
+        },
+        {
+            "id": "5934E4C6-7AE4-46A4-8F92-99101987D064",
+            "use": "show_icon_on_start",
+            "template": "Show an icon on the LED screen when the program starts",
+            "description": "An icon block is used inside the on start block",
+            "hideInCatalog": true,
+            "docPath": "/teachertool"
+        },
+        {
+            "id": "0B2BC680-D79D-42C8-BA13-B6B189E2BF7D",
+            "use": "point_condition",
+            "template": "If an LED point is truthy, then uplot that point and plot a new point",
```

I think we would benefit from using more casual, user-oriented language. Some of these are a bit too javascripty, I feel.

```suggestion
            "template": "If an LED point is lit, then unplot that point and plot a new point",
```

---

ditto for the description here

---

"If an LED is on, turn it off and light up a different LED."

---

I think we still need to update this one. I like Eric's suggestion.